src/promptflow/tests/sdk_cli_test/unittests/test_exceptions.py

```diff
@@ -62,9 +78,9 @@ def test_error_category_with_http_error(self, subscription_id, resource_group_na
         assert error_target == ErrorTarget.UNKNOWN
         assert error_message == ""
         assert (
-            "exception module=Non promptflow module, not recorded., "
-            "exception code=Non promptflow code, not recorded., "
-            "exception lineno=Non promptflow code lineno, not recorded."
+            "module=Non promptflow module, not recorded, "
```

Comment:
how about make this empty to save our network bandwidth?

Reply:
I think so, I have changed it to empty.